examples/monitor-with-externalConfigMap/external-configMap.yaml

```diff
@@ -0,0 +1,164 @@
+# Source: tidb-cluster/templates/monitor-configmap.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: external-config
+  labels:
+    app.kubernetes.io/name: tidb-cluster
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/instance: test
+    app.kubernetes.io/component: monitor
+    helm.sh/chart: tidb-cluster-dev
```

Comment:
I think we can remove `app.kubernetes.io/managed-by: Helm` and `helm.sh/chart: tidb-cluster-dev` in this example

Reply:
done